src/CSS/footer.css

```diff
@@ -0,0 +1,26 @@
+footer {
+  margin-top: 3rem;
+  width: 100%;
+  height: 100px;
+  background-color: purple;
+  color: white;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  font-size: 20px;
+  font-weight: bolder;
+  position: relative;
+}
```

Comment:
- [ ] Kindly make sure you stick to the design as much as possible. I mean even you will have to personalize your styles which is great, the positions of elements are supposed to be similar to the design.
- [ ] You can add a `thumb-up` icon also instead of a generic `text` for the likes.
Your design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/197340281-8030b4f6-3c12-4e7b-b105-f7459e7c5b89.png)
The provided design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/197340302-a469dcf6-a069-432c-90c8-012d730c160f.png)